cmd/transflect-operator/operator.go

```diff
@@ -68,26 +73,51 @@ func newOperator(cfg *config) (*operator, error) {
 		k8s:   k8s,
 		istio: istio,
 
-		useIngress: cfg.UseIngress,
-		plaintext:  cfg.Plaintext,
-		address:    cfg.Address,
-		version:    "transflect-" + version,
+		useIngress:      cfg.UseIngress,
+		plaintext:       cfg.Plaintext,
+		address:         cfg.Address,
+		version:         "transflect-" + version,
+		leaderNamespace: cfg.LeaseNamespace,
```

Comment:
nit: i think this should also be called `leaseNamespace` not `leaderNamespace`, the same as it is in the config.

Or if you don't want "lease" in the same, since the lock could be a different type of resource, you could call both "leaderLockNamespace". But meh.

Reply:
leaseNamesapce it is, fixed.